### PR TITLE
Added International Airport codes

### PR DESCRIPTION
**Pull request by mackenza**

### CODE REVIEW

**Comment by vlucas** [comment]
I love this data set. Is there a quick way to make the key be the airport code, and move the name into a "name" key on the object? The reason is that people are going to want to store the airport code in their database, not the name, and that is easier if the value is in the key (when put into select boxes, etc.).

``` json
{
  "YXX": {
    "country":"Canada",
    "name":"Abbotsford, BC"
  },

  // etc...
```

**Comment by mackenza** [comment]
yeah, I thought the same. Let me revise it.

File: app.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ var datasets = {
   'states': 'U.S. States',
   'canadian-provinces': 'Canadian Provinces',
   'countries': 'World Countries',
-  'currencies': 'World Currencies'
+  'currencies': 'World Currencies',
+  'airportcodes': 'International Airport Codes'
```
Comment (mikealmond) [inline app.js]: `airportcodes` => `airport-codes` for consistency?

Comment (vlucas) [inline app.js]: Yeah, go ahead and update that to match.

**Comment by mackenza** [comment]
I made those changes... it says there is a merge conflict but I don't know what that would be

**Comment by mackenza** [comment]
oh... maybe I need to rebase my fork

**Comment by mackenza** [comment]
that worked

**Comment by vlucas** [comment]
Did you mean to delete the timezones dataset? That one was already merged in.

**Comment by mackenza** [comment]
sigh no... git noob stuff, I guess... I am going to delete this PR and try again.

**Pull request closed without merging**